packages/expo-system-ui/unimodule.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "expo-module-template",
```

This name is in fact not really needed, but let's make it correct.
```suggestion
  "name": "expo-system-ui",
```

Also, since it's going to be a Swift module, you can rename the config file to `expo-module.config.json` which is the new name for it. To link the Swift module properly, you'll also need to add:

```
"ios": {
  "modulesClassNames": ["SystemUIModule"]
}
```

There is a plan to detect this automatically by `expo-modules-autolinking`, but for now we need to be explicit (it's also faster because we don't have to traverse the file tree to find it)